### PR TITLE
Removes #basemaps

### PR DESCRIPTION
**Pull request by iriberri**
Fixes https://github.com/CartoDB/cartodb/issues/3855

@javisantana

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3992/))

**Comment by javisantana** [comment]
please @alonsogarciapablo confirm the submodule commit is compatible with cartodb

@iriberri in order to change that in production you should change in chef cookbooks

**Comment by iriberri** [comment]
Fixing the cookboks right now

File: lib/assets/javascripts/cdb [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 9c9343eba77e465e4c865a7a9a5d01567909680a
+Subproject commit f73f90c1ae5e00df6eb8555da7609a840380d1c0
```
Comment (alonsogarciapablo) [inline lib/assets/javascripts/cdb]: Why are we changing this? I'm working on releasing cartodb.js v3.14.3, so this will soon be pointing to this version.

Comment (iriberri) [inline lib/assets/javascripts/cdb]: Not a special reason, it happened when I tried to push the changes because I don't know how submodules work

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/3993/))

**Comment by alonsogarciapablo** [comment]
:+1:

**Comment by saleiva** [comment]
bump!

**Pull request merged**